### ISSUE TITLE
3.5.3 crashes when run

### ISSUE THREAD

**Issue by mattiaswerner** [glasen/snap-nextcloud-desktop-client#9]
## Bug description
After the most recent update to the snap, the desktop client fails to run.

## Steps to reproduce

```
mattias@calliope:~$ nextcloud-desktop-client.nextcloud
Warning: Ignoring XDG_SESSION_TYPE=wayland on Gnome. Use QT_QPA_PLATFORM=wayland to run on Wayland anyway.
Qt: Session management error: Could not open network socket
Aborted (core dumped)

```
## Expected behaviour
Client to run without issue

## Operating system
Ubuntu 22.04 fully updated

## Nextcloud Desktop Client version
3.5.3

## Additional info
Reproduced issue on two different systems.
Same issue on Xorg and Wayland desktops.
Using `snap revert` to previous version (70) works fine.

**Comment by glasen** [comment]
Can you please stop the client, upgrade to the latest revision, delete the directory "~/snap/nextcloud-desktop-client/common/.cache" and try to start the latest revision again?

**Comment by mattiaswerner** [comment]
> Can you please stop the client, upgrade to the latest revision, delete the directory "~/snap/nextcloud-desktop-client/common/.cache" and try to start the latest revision again?

Done. Same issue.

**Comment by glasen** [comment]
I've reverted the snap to the core20-runtime. I hope this fixes the problem.

**Comment by mattiaswerner** [comment]
Just tried it. Same issue. Reverting back to (70) makes it work again.

**Comment by glasen** [comment]
I've made a mistake reverting to core20. I'll currently building the real core20-based snap. It should be ready in about 20min.

**Comment by glasen** [comment]
The snap is ready. The revision is 82. The arm64-snap is currently building.

**Comment by mattiaswerner** [comment]
Updated. It runs, only now the client is reporting sync issues as follows
![image](https://user-images.githubusercontent.com/2061171/186905661-106c8240-de10-4fb1-8537-23d13b10d4b2.png)

**Comment by glasen** [comment]
You must (!) stop the client and restart it. The directory the client mentions gets deleted on the update. All content from it gets moved to "82/Nextcloud". If you have stored anything in this folder, you should move it to another place and make sync from this folder. Never use a folder from the "~/snap/nextcloud-desktop-client"-folder for storing your files! These are only for config, log and other files for the client.

**Comment by mattiaswerner** [comment]
I always stop the client before I update the snap.

On my laptop I got the sync issues, resolved it by removing the sync directory in the client and re-adding by pointing it to "~/snap/nextcloud-desktop-client/current/Nextcloud"

On my desktop I was still getting the same segmentation fault. Resolved it by manually deleting the '70' directory in the "~/snap/nextcloud-desktop-client/" folder and it launches again just fine. Re-added the sync folder pointing at the 'current' directory.
